src/pages/patientView/PatientViewPage.tsx

```diff
@@ -43,6 +45,7 @@ interface IPatientViewState {
     mutationData: any;
     mrnaExprRankData?: MrnaRankData;
     activeTabKey: number;
+    hotspotsData: any;
```

Comment:
the vdom will always be re-generated whenever there's a change to data either way.